docs/create_dimensions.md

```diff
@@ -3,7 +3,7 @@
 In Maximo Asset Monitor, dimensions are a way of aggregating your sensor data in different hierarchical categories and locations. Using a REST API call, you are able to push JSON into the Monitor data lake to assign dimensions to different sensor device IDs.  
 
 Let’s look at an application of dimensions in Monitor. Dimensions in Maximo Asset Monitor allow for rapid actionable insights. The Workstation demo showcases the IBM IoT Center in Munich which has been outfitted with sensors to measure comfort and occupancy. Our developers added the following dimensions: Region, Building, Floor, Zone, and Workstation. Right when you log in to the Building Summary dashboard, you can see on the alerts table that all 120 alerts for October 10 came from the Munich HQ building on the 27th floor in Zone 3. The dimensions allow you to tie your sensors with a specific location which allows you to be able to drill down into Zone 3 inside of the dashboard or physically go to Zone 3 and check out the problem.  
```

Comment:
Not everyone knows what "workstation demo  is". We need to make this generic to people at partner sites that haven't seen all the Monitor-Demo environment entity types and dashboards.